.vue.webpack.config.js

```diff
@@ -0,0 +1,62 @@
+const webpack = require('webpack');
+const path = require('path');
+const VueLoaderPlugin = require('vue-loader').VueLoaderPlugin;
+
+const config = {
+    entry: {
+        'vue': './js/src/app.js',
```

Comment:
Maybe just move the source file in a `vue` sub-directory, so that the `public/build` directory structure would be some kind of mirror of the `js/src` directory.

```suggestion
        'vue': './js/src/vue/app.js',
```